cmd/ttn-lw-cli/commands/use.go

```diff
@@ -25,23 +25,42 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 	"go.thethings.network/lorawan-stack/v3/pkg/errors"
+	"go.thethings.network/lorawan-stack/v3/pkg/log"
 	"go.thethings.network/lorawan-stack/v3/pkg/rpcmiddleware/discover"
 	"gopkg.in/yaml.v2"
 )
 
 const configFileName = ".ttn-lw-cli.yml"
 
+func useCommandWriteConfigFlags() *pflag.FlagSet {
+	flagSet := &pflag.FlagSet{}
+	flagSet.Bool("user", false, "Write config file in user config directory")
+	flagSet.Bool("overwrite", false, "Overwrite existing config files")
```

Comment:
I suppose you can move these flags back to the bottom now that we're back to a single command